### DISCUSSION TITLE
Support Touch Screen

### DISCUSSION THREAD

**Discussion by nguyenanhgiau** [GloDroid/glodroid_manifest#190] [Ideas]
Hi all,
I try to user GloDroid project with a raspberry pi4 4G RAM and a touch screen [7inch Capacitive Touch Screen](https://www.waveshare.com/7inch-hdmi-lcd-h-with-case.htm)
But the device didn't boot to home screen.

As I know, GloDroid project has not supported this touch screen. 
How to support this monitor?
More information:
GloDroid ver: v0.7.1

Thanks a lot!

**Comment by rsglobal** [comment]
Hi,

Please check and confirm that this touch screen works well with raspbian on the same board.

**Reply by nguyenanhgiau** [reply]
Hi @rsglobal 
yes, I checked this touch screen with raspbian OS and it worked well.

**Reply by rsglobal** [reply]
Nice, could you attach logs (dmesg and logcat) after Android boot?

```
adb root
adb logcat | tee logcat.txt
adb shell dmesg | tee dmesg.txt
adb shell dumpsys SurfaceFlinger | tee ds_sf.txt
```

**Reply by nguyenanhgiau** [reply]
Hi @rsglobal
I'm so sorry about this topic.
This touch screen didn't work at the first time, so I have to reboot my device.
And now, it is working very well.
Thanks a lot!
